docs/reference/technology/technology.md

```diff
@@ -1,3 +1,5 @@
 # Technology
 
 This section of the book talks about technologies that Mbed OS implements or supports. The content in this section details the architecture and protocols, design decisions and historical context of these technologies.
+
+<!---Should we have links to the subsections here?--->
```

Comment:
This section is quite empty, and I'd rather combine it with an index of the pages below.

Reply:
I'm going to remove this comment for now. We can address this page in another PR.